web/controllers/certificateController/index.ts

```diff
@@ -3,7 +3,7 @@ import { Request, Response } from 'express';
 import { Pupil } from '../../../common/entity/Pupil';
 import { Student } from '../../../common/entity/Student';
 import { getTransactionLog } from '../../../common/transactionlog';
-import { getManager } from 'typeorm';
+import { getManager, getRepository } from 'typeorm';
```

Comment:
Looks like this import is unused 🤭 (But it's not the only one in this file and in the entire code, so not really important)